### DISCUSSION TITLE
Multiple Users?

### DISCUSSION THREAD

**Discussion by levnorth** [givanz/Vvveb#149] [General]
I would like to know if there is a multiple user feature where different users can log in, create posts, etc.

**Comment by givanz** [comment]
[Admin users](https://demo.vvveb.com/admin/index.php?module=admin/users) can have different roles that can limit access to features, for example users with [author role](https://demo.vvveb.com/admin/index.php?module=admin/user&admin_id=5) can only add/edit posts/pages and access page builder.

https://demo.vvveb.com/admin/index.php?module=admin/roles

Regular users/customers can only post comments/reviews or place orders etc https://demo.vvveb.com/admin/index.php?module=user/user&user_id=1
